src/global.css

```
@@ -8,6 +8,8 @@ html {
   height: 100%;
   font-size: 18px;
   -webkit-font-smoothing: antialiased;
+  scroll-behavior: smooth;
+  scroll-padding-top: calc(var(--header-height) * 1.4);
 }
 
```

Just in case our accessibility expert sees this code.

```suggestion

@media (prefers-reduced-motion) {
  html {
    scroll-behavior: auto;
  }
}

```